src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/SinglePhaseReservoirFVM.hpp

```diff
@@ -59,13 +59,16 @@ class SinglePhaseReservoirFVM : public MGRStrategyBase< 1 >
     setupLabels();
 
     // Level 0
-    m_levelFRelaxMethod[0]     = MGRFRelaxationMethod::singleLevel;
-    m_levelFRelaxType[0]       = MGRFRelaxationType::gsElimWInverse; //default, i.e. Jacobi
+#ifdef GEOSX_USE_HYPRE_CUDA
+    m_levelFRelaxType[0]          = MGRFRelaxationType::l1jacobi;
+#else
+    m_levelFRelaxType[0]          = MGRFRelaxationType::gsElimWInverse;
+#endif
```

Comment:
For all the recipes with wells, instead of changing the option for the CUDA builds, I would be in favor of throwing an error here to terminate the simulation on GPU. Just to discourage the use of the recipe while `gsElimWInverse` is not working on GPU. Advanced users can always recompile if they want to use this recipe on GPU.

Reply:
Agreed

Reply:
Agree. @victorapm this has been fixed in the latest version of _hypre_, correct?

Reply:
Not yet. However, we do have ILU support with CUDA/HIP ready on hypre/master

Reply:
Gaussian elimination is now allowed exclusively when using CPU (`GEOS_USE_HYPRE_CPU`).